osu.Game/Beatmaps/Drawables/BeatmapBackgroundSprite.cs

```diff
@@ -23,8 +23,9 @@ public BeatmapBackgroundSprite(IWorkingBeatmap working)
         [BackgroundDependencyLoader]
         private void load()
         {
-            if (working.Background != null)
```

Comment:
This example alone should be enough for anyone to jump on this change. It's actually retrieving two backgrounds (one in the null check).